grafana/overlays/nerc-ocp-infra/kustomization.yaml

```
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../base
+  - externalsecrets
+  - patches/logging-grafana-patch.yaml
+
+patchesStrategicMerge:
+  - grafanas/logging-grafana_patch.yaml
```

This should be:

```suggestion
patches:
- path: grafanas/logging-grafana_patch.yaml
```

`patchesStrategicMerge` has been [deprecated for a while](https://github.com/kubernetes-sigs/kustomize/issues/5052).